pkg/provider/vault/vault_test.go

```diff
@@ -1166,10 +1166,13 @@ func TestGetAllSecrets(t *testing.T) {
 func TestGetSecretPath(t *testing.T) {
 	storeV2 := makeValidSecretStore()
 	storeV2NoPath := storeV2.DeepCopy()
+	multiPath := "secret/path"
```

Comment:
You changed this (for a good reason) in this PR. I quickly checked that the old tests with `args.path` and `args.expected` would still work. :+1:
So this should not be a breaking change